config/acer-a1-724.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+# DTB: From Acer_AV0K1_A1-724_RV04RC07_WW_GEN1.zip
+# https://androidfilehost.com/?fid=889764386195913456
+
+OPTIONS=(--dcs-no-get-brightness)
+PANELS=(
+	[hx8394d_720p_video]="acer,hx8394d"
```

Comment:
This feels a bit too generic, maybe

```suggestion
	[hx8394d_720p_video]="acer,a1-724-hx8394d"
```